src/app/commons/Spinner.scss

```diff
@@ -0,0 +1,88 @@
+.sk-folding-cube {
```

Comment:
this should be a `css` file since

Reply:
I changed it to `scss` style when you are reviewing. 😄